File: android/app/src/main/java/com/example/demo/data/SubscriptionPlan.kt
package com.example.demo.data

data class SubscriptionPlan(
    val name: String,
    val price: String,
    val features: List<String>,
    val isHighlighted: Boolean = false,
    val isEnabled: Boolean = true,
)

data class PlanConfiguration(
    val showEnterprisePlan: Boolean = false,
    val enterprisePrice: String = "-",
    val highlightPremium: Boolean = false,
)

File: ios/ConfidenceDemoApp/Models/SubscriptionPlan.swift
import Foundation

struct SubscriptionPlan: Identifiable, Equatable {
    let id = UUID()
    let name: String
    let price: String
    let features: [String]
    let isHighlighted: Bool
    let isEnabled: Bool

    init(name: String, price: String, features: [String], isHighlighted: Bool = false, isEnabled: Bool = true) {
        self.name = name
        self.price = price
        self.features = features
        self.isHighlighted = isHighlighted
        self.isEnabled = isEnabled
    }
}

struct PlanConfiguration {
    let showEnterprisePlan: Bool
    let enterprisePrice: String
    let highlightPremium: Bool

    init(showEnterprisePlan: Bool = false, enterprisePrice: String = "-", highlightPremium: Bool = false) {
        self.showEnterprisePlan = showEnterprisePlan
        self.enterprisePrice = enterprisePrice
        self.highlightPremium = highlightPremium
    }
}